cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -578,13 +578,16 @@ def qscout_compile(
     def cq_compile(
         self,
         circuits: Union[cirq.Circuit, Sequence[cirq.Circuit]],
+        grid_shape: Optional[Tuple[int, int]] = None,
         target: str = "cq_hilbert_qpu",
```

Comment:
nit
```suggestion
        target: str = "cq_hilbert_qpu",
        *,
        grid_shape: Optional[Tuple[int, int]] = None,
```
(this will force users to pass `grid_shape=` explicitly (e.g. `cq_compile(circuits, grid_shape=(4, 5))`) instead of just using `cq_compile(circuits, (4, 5))`, which prevents us from breaking anyone's code if/when we add more arguments or move them around)